extensions/amp-subscriptions/0.1/viewer-subscription-platform.js

```diff
@@ -77,16 +77,26 @@ export class ViewerSubscriptionPlatform {
   /** @override */
   getEntitlements() {
     devAssert(this.currentProductId_, 'Current product is not set');
-
+    /** @type {Object} */
+    let messageData = dict({
+      'publicationId': this.publicationId_,
+      'productId': this.currentProductId_,
+      'origin': this.origin_,
+    });
+    const encryptedDocumentKey =
+        this.serviceAdapter_.getEncryptedDocumentKey('google');
```

Comment:
I think it's `google.com`. But more importantly we don't know which viewer it actually is. We need to check the `viewerUrl` to know, or more specifically we need to iterate via configured platforms and check whether any of these support the viewer. Let's fix the code, but we can just add TODO for now for the viewer matching.